app/controllers/admin/whitelists_controller.rb

```diff
@@ -21,6 +21,15 @@ def create
         presenter: UseCases::Administrator::CreateSignupWhitelist.new
```

Comment:
I think we should rename this to be more different than the new one. Eg: `FormatEmailWhitelistRegex`